### PR TITLE
Cosmos 2671: Debug the issue

### PR DESCRIPTION
**Pull request by LokeshvarKr**
Cosmos 2671: Debug the issue

### CODE REVIEW

File: compiler/pkg/generator/template/client.go.tmpl [hunk 1]
```diff
@@ -146,47 +146,69 @@ func new{{$group.GroupTypeName}}(client *Clientset) *{{$group.GroupTypeName}} {
...
+		}
+		break
+	}
+
+	if cache_obj !=nil {
+		return cache_obj,nil
+	}
+	if api_obj!=nil{
+		return api_obj,nil
 	}
```
Comment (xmen4xp) [inline compiler/pkg/generator/template/client.go.tmpl:210]: The api_obj should not be returned ?
It is only useful for debugging to check what is the the DB vs what is in the cached version of the same object.

Comment (LokeshvarKr) [reply compiler/pkg/generator/template/client.go.tmpl:210]: Sure @xmen4xp

Comment (LokeshvarKr) [reply compiler/pkg/generator/template/client.go.tmpl:210]: I will make the change

**Comment by LokeshvarKr** [comment]
This was just to debug and test.

**Pull request closed without merging**